server/Transaction.java

```diff
@@ -229,12 +247,12 @@ private boolean checkDouble(String s) {
     }
 
     // Returning true if there are sufficient funds, else false.
-    private boolean checkSufficientFunds(double amount, Account fromAccount) {
+    private boolean checkSufficientFunds(int amount, Account fromAccount) {
         return !((fromAccount.getBalance() - amount) < 0);
```

Comment:
That's a really interesting way of writing this, took me a bit to realize what was the point!
Doesn't `fromAccount.getBalance() - amount > 0` feel more natural?